File: Java/pares_consecutivos.java
import java.util.Locale;
import java.util.Scanner;

public class pares_consecutivos {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int x, total;

		total = 0;

		System.out.print("Digite um numero inteiro: ");
		x = sc.nextInt();

		while (x != 0) {
			if (x % 2 != 0) {
				x = x + 1;
			}

			total = 5 * x + 20;
			System.out.println("SOMA = " + total);

			System.out.print("Digite um numero inteiro: ");
			x = sc.nextInt();
		}
		sc.close();
	}
}

File: C/pares_consecutivos.c
#include <stdio.h>

int main()
{
    int x, soma;

    printf("Digite um numero inteiro: ");
    scanf("%d", &x);

    while (x != 0){
      if (x % 2 != 0){
        x = x + 1;
      }

      soma = 0;
      for (int i = 0; i < 5; i++){
        soma = soma + x;
        x = x + 2;
      }
      //Ou substituindo o for: soma = 5 * x + 20;

      printf("SOMA = %d\n", soma);

      printf("Digite um numero inteiro: ");
      scanf("%d", &x);
    }

    return 0;
}
